src/commands/task/delete.ts

```diff
@@ -61,10 +65,10 @@ export default class TaskDeleteCommand extends Command {
       if (answer) {
         if (taskId) {
           const success = await this.relay.deleteTask(subscriberId, taskEndpoint, taskId)
-          success ? this.log(`Task deleted`) : this.log(`Task NOT deleted`)
+          success ? this.log(`Task deleted`) : this.error(`Task NOT deleted, make sure task exists (relay tasks list)`)
         }
         else if (flags.tag) {
-          let tasks = await this.relay.fetchTasks(subscriberId, taskEndpoint)
+          let tasks = await this.relay.fetchTasks(subscriberId, taskEndpoint, ``)
```

Comment:
Is this correct? Seems the path won't construct correctly.

On the other hand, If you want to have an optional parameter with a default (like an empty string):

```
async fetchTasks(subscriberId: string, groupId: string, taskEndpoint=``): Promise<Task[]> {
```

And then you can use like:

```
fetchTasks(subscriberId, taskEndpoint)
```

Reply:
Whoops yes groupId and taskEndpoint should be swapped in the function declaration, will change to 
`async fetchTasks(subscriberId: string, taskEndpoint: string, groupId=''`